sympy/concrete/tests/test_sums_products.py

```diff
@@ -1133,12 +1133,9 @@ def test_issue_14640():
 
 
 def test_issue_15943():
-    n, k = symbols('n k', integer=True, nonnegative=True)
-    s = Sum(binomial(n, k)*factorial(n - k), (k, 0, n)).doit().rewrite(gamma)
-    assert s == -E*(n + 1)*gamma(n + 1)*lowergamma(n + 1, 1)/gamma(n + 2
-        ) + E*gamma(n + 1)
-    assert s.simplify() == E*(factorial(n) - lowergamma(n + 1, 1))
-
+    n = Symbol('n', integer=True, nonnegative=True)
+    s = Sum(binomial(n, k)*factorial(n - k), (k, 0, n)).doit()
+    assert s == E*factorial(n)
```

Comment:
https://www.wolframalpha.com/input/?i=Sum%28binomial%28n%2C+k%29*factorial%28n+-+k%29%2C+%28k%2C+0%2C+n%29%29

I checked on WolframAlpha.
It gave the answer as `e Γ(1 + n, 1)`
which is equivalent to `e*factorial(n)` right ?